### ISSUE TITLE
Package.getActivationCommands is deprecated.

### ISSUE THREAD

**Issue by prometheas** [190n/open-doc#3]
This guy, too :)

Use `activationCommands` instead of `activationEvents` in your package.json
Commands should be grouped by selector as follows:

``` json
  "activationCommands": {
    "atom-workspace": ["foo:bar", "foo:baz"],
    "atom-text-editor": ["foo:quux"]
  }
```

```
Package.getActivationCommands (/Users/lianoglj/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:828:9)
Package.hasActivationCommands (/Users/lianoglj/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:748:20)
Package.activationShouldBeDeferred (/Users/lianoglj/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:738:19)
<unknown> (/Users/lianoglj/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:189:55)
Package.measure (/Users/lianoglj/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:167:15)
Package.load (/Users/lianoglj/Applications/Atom.app/Contents/Resources/app.asar/src/package.js:181:12)
```

**Comment by 190n** [comment]
Sorry @prometheas, I wrote this a while ago and I've been lazy about upgrading for 1.0.

**Comment by 190n** [comment]
Fixed in v0.3.5 (a832f13).
